### PR TITLE
Spring 04

### PR DESCRIPTION
**Pull request by Alex-Korzun**

### CODE REVIEW

**Review by Likh-Alex — APPROVED** [review]

File: src/main/java/com/dev/cinema/config/AppConfig.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@PropertySource("classpath:db.properties")
+@ComponentScan(basePackages = {
+        "com.dev.cinema.service",
+        "com.dev.cinema.dao"
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/dev/cinema/config/AppConfig.java]: security

File: src/main/java/com/dev/cinema/controller/HelloController.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.dev.cinema.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class HelloController {
+    @ResponseBody
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/dev/cinema/controller/HelloController.java]: let's use RestController

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/java/com/dev/cinema/config/AppConfig.java (hunk 1 above)
Comment (Alex-Korzun) [reply src/main/java/com/dev/cinema/config/AppConfig.java]: You mean Authentication services move to Security package?

Comment (Alex-Korzun) [reply src/main/java/com/dev/cinema/config/AppConfig.java]: done

File: src/main/java/com/dev/cinema/controller/HelloController.java (hunk 2 above)
Comment (Alex-Korzun) [reply src/main/java/com/dev/cinema/controller/HelloController.java]: done

File: src/main/java/com/dev/cinema/controller/HelloController.java [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 
-import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
 
-@Controller
+@RestController
 public class HelloController {
     @ResponseBody
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/dev/cinema/controller/HelloController.java]: do we need this annotation now? What is the difference between @Controller and @RestController?

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/java/com/dev/cinema/controller/HelloController.java (hunk 3 above)
Comment (Alex-Korzun) [reply src/main/java/com/dev/cinema/controller/HelloController.java]: sorry was inattentive
@RestController already contains @ResponseBody in it

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**